### ISSUE TITLE
sohu video cannot be downloaded，because of "Forbidden"

### ISSUE THREAD

**Issue by JiaxiangBU** [soimort/you-get#1979]
Please make sure these boxes are checked before submitting your issue – thank you!

- [ ] You can actually watch the video in your browser or mobile application, but not download them with `you-get`.
- [ ] Your `you-get` is up-to-date.
- [ ] I have read <https://github.com/soimort/you-get/wiki/FAQ> and tried to do so.
- [ ] The issue is not yet reported on <https://github.com/soimort/you-get/issues> or  <https://github.com/soimort/you-get/wiki/Known-Bugs>. If so, please add your comments under the existing issue.
- [ ] The issue (or question) is really about `you-get`, not about some other code or project.

Run the command with the `--debug` option, and paste the full output inside the fences:

```
you-get --debug 'http://tv.sohu.com/20170515/n493147782.shtml'
[DEBUG] get_response: http://tv.sohu.com/20170515/n493147782.shtml
[DEBUG] get_response: http://hot.vrs.sohu.com/vrs_flash.action?vid=3736313
[DEBUG] get_response: http://hot.vrs.sohu.com/vrs_flash.action?vid=3736316

[DEBUG] get_response: http://121.46.19.92/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170512/89093558_3736315_tv_S_210230_31571/89093558_3736315_tv_S_210230_31571_001.mp4&new=/89/210/ULy94qEbpQTWM3qDRAz8eF.mp4&key=jRrqXnwGpUBm6m1gmyI5oogeTMtefu3A&vid=3736316&uid=1494867928382&t=0.572417551461445&rb=1
[DEBUG] get_response: http://121.46.19.92/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170512/89093558_3736315_tv_S_210230_31571/89093558_3736315_tv_S_210230_31571_002.mp4&new=/75/34/s3Y0SHoYSBGX6MAJhifAJB.mp4&key=y88DQOI-p710Dgy2j1awNtv9Ef3ayT2O&vid=3736316&uid=1494867928493&t=0.3359180173885149&rb=1
[DEBUG] get_response: http://121.46.19.92/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170512/89093558_3736315_tv_S_210230_31571/89093558_3736315_tv_S_210230_31571_003.mp4&new=/2/75/3pLyeXifStuyrbYgby7dHB.mp4&key=nMW4BDu6eokRnJc-dfGwjd--Mr5-hCvf&vid=3736316&uid=1494867928592&t=0.7101166756626843&rb=1
[DEBUG] get_response: http://121.46.19.92/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170512/89093558_3736315_tv_S_210230_31571/89093558_3736315_tv_S_210230_31571_004.mp4&new=/52/206/nGaFdsqDSYSchTlWOUXRsB.mp4&key=GbJFkzmX_H-yNh1-CIMQg8EcxHJya195&vid=3736316&uid=1494867928660&t=0.6991724572006472&rb=1
[DEBUG] get_response: http://121.46.19.92/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170512/89093558_3736315_tv_S_210230_31571/89093558_3736315_tv_S_210230_31571_005.mp4&new=/53/60/jQIZDelfQUeM0qLGJKjZEC.mp4&key=8ixKBJdO0wVsLINIorhMS4irgUD3qu2u&vid=3736316&uid=1494867928729&t=0.9856412343707734&rb=1
[DEBUG] get_response: http://121.46.19.92/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170512/89093558_3736315_tv_S_210230_31571/89093558_3736315_tv_S_210230_31571_006.mp4&new=/130/107/GvNjfNqiSYSsc9ZbJFmFtF.mp4&key=NIzxoqXSHvhallkE1AlilKUUQ6lhpQUq&vid=3736316&uid=1494867928811&t=0.41727348164204314&rb=1
[DEBUG] get_response: http://121.46.19.92/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170512/89093558_3736315_tv_S_210230_31571/89093558_3736315_tv_S_210230_31571_007.mp4&new=/174/83/eA3z3fZvR8fUyur0Y6sRjB.mp4&key=NsIEtS0cVSQmcmTIggyQSUvNOgZQTMfR&vid=3736316&uid=1494867928898&t=0.19772057765556572&rb=1
[DEBUG] get_response: http://121.46.19.92/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170512/89093558_3736315_tv_S_210230_31571/89093558_3736315_tv_S_210230_31571_008.mp4&new=/14/202/D9DNu5F7oxJA19yyNvDJOI.mp4&key=lsLu26VvGIJJN7ziJqqBx_U6VN_vqqsS&vid=3736316&uid=1494867929009&t=0.03718431687959989&rb=1
[DEBUG] get_response: http://121.46.19.92/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170512/89093558_3736315_tv_S_210230_31571/89093558_3736315_tv_S_210230_31571_009.mp4&new=/54/196/37Kwl3P7TWaZSD2odB95DC.mp4&key=egC4B-6jLbZTXRYF_EQ7FpSJ8uHmCYux&vid=3736316&uid=1494867929092&t=0.40975078461825665&rb=1
Site:       Sohu.com
Title:      欢乐颂2第6集
Type:       MPEG-4 video (video/mp4)
Size:       313.21 MiB (328419483 Bytes)

Downloading 欢乐颂2第6集.mp4 ...
 0.0% (  0.0/313.2MB) ├─────────────────────────────────────────┤[1/9] you-get: version 0.4.715, a tiny downloader that scrapes the web.
you-get: ['http://tv.sohu.com/20170515/n493147782.shtml']
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.6/bin/you-get", line 11, in <module>
    load_entry_point('you-get==0.4.715', 'console_scripts', 'you-get')()
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/__main__.py", line 92, in main
    main(**kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/common.py", line 1443, in main
    script_main('you-get', any_download, any_download_playlist, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/common.py", line 1354, in script_main
    download_main(download, download_playlist, args, playlist, output_dir=output_dir, merge=merge, info_only=info_only, json_output=json_output, caption=caption)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/common.py", line 1156, in download_main
    download(url, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/common.py", line 1436, in any_download
    m.download(url, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/extractors/sohu.py", line 75, in sohu_download
    download_urls(urls, title, 'mp4', size, output_dir, refer = url, merge = merge)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/common.py", line 810, in download_urls
    url_save(url, filepath, bar, refer = refer, is_part = True, faker = faker, headers = headers, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/common.py", line 531, in url_save
    response = urlopen_with_retry(request.Request(url, headers=headers))
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/you_get/common.py", line 306, in urlopen_with_retry
    return request.urlopen(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 223, in urlopen
    return opener.open(url, data, timeout)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 532, in open
    response = meth(req, response)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 642, in http_response
    'http', request, response, code, msg, hdrs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 570, in error
    return self._call_chain(*args)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 504, in _call_chain
    result = func(*args)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 650, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

If there's anything else you would like to say (e.g. in case your issue is not about downloading a specific video; it might as well be a general discussion or proposal for a new feature), fill in the box below; otherwise, you may want to post an emoji or meme instead:

> [WRITE SOMETHING]
> [OR HAVE SOME :icecream:!]

汉语翻译最终日期：2016年02月26日

在提交前，请确保您已经检查了以下内容!

- [ ] 你可以在浏览器或移动端中观看视频，但不能使用`you-get`下载.
- [ ] 您的`you-get`为最新版.
- [ ] 我已经阅读并按 <https://github.com/soimort/you-get/wiki/FAQ> 中的指引进行了操作.
- [ ] 您的问题没有在<https://github.com/soimort/you-get/issues> , <https://github.com/soimort/you-get/wiki/FAQ> 或 <https://github.com/soimort/you-get/wiki/Known-Bugs> 报告，否则请在原有issue下报告.
- [ ] 本问题确实关于`you-get`, 而不是其他项目.

请使用`--debug`运行，并将输出粘贴在下面:

```
[在这里粘贴完整日志]
```

如果您有其他附言，例如问题只在某个视频发生，或者是一般性讨论或者提出新功能，请在下面添加;或者您可以卖个萌:

> [您的内容]
> [舔 :icecream:!]

**Comment by rosynirvana** [comment]
试了一下是因为sohu的CDN不认 ```Range: bytes=0-```，莫名其妙

**Comment by rosynirvana** [comment]
fixed via dbe272a

```
$ ./you-get -d 'http://tv.sohu.com/20170515/n493147782.shtml'
[DEBUG] get_response: http://tv.sohu.com/20170515/n493147782.shtml
[DEBUG] get_response: http://hot.vrs.sohu.com/vrs_flash.action?vid=3736313
[DEBUG] get_response: http://hot.vrs.sohu.com/vrs_flash.action?vid=3736316
[DEBUG] get_response: http://121.46.19.90/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170606/89093558_3736315_tv_S_114528_26293/89093558_3736315_tv_S_114528_26293_001.mp4&new=/165/168/CBFq8SPtJg8mxQ96uJkyZa.mp4&key=gushPWH9zKwgI2AIKtl2hvvKXBOSu3r4&vid=3736316&uid=1499605810444&t=0.02273481206728767&rb=1
[DEBUG] get_response: http://121.46.19.90/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170606/89093558_3736315_tv_S_114528_26293/89093558_3736315_tv_S_114528_26293_002.mp4&new=/239/104/wrHleg80oHnNAJS82BC2xB.mp4&key=NV3Lj2ERgG_zPV0ziyqgk77F1__uRyux&vid=3736316&uid=1499605810545&t=0.2466364734464802&rb=1
[DEBUG] get_response: http://121.46.19.90/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170606/89093558_3736315_tv_S_114528_26293/89093558_3736315_tv_S_114528_26293_003.mp4&new=/29/47/KNL4ad8TI9BqzFtuYqhA1F.mp4&key=YivBJWWEQW-3QXvaXs-GoJ2yMGF80cOW&vid=3736316&uid=1499605810647&t=0.8386525250624308&rb=1
[DEBUG] get_response: http://121.46.19.90/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170606/89093558_3736315_tv_S_114528_26293/89093558_3736315_tv_S_114528_26293_004.mp4&new=/45/75/9HtqBj6RIaRWkMPdVZV4tG.mp4&key=Z34fcCMT1cdmSa84ilmqXkNFVIDyXaY4&vid=3736316&uid=1499605810719&t=0.8442468505156223&rb=1
[DEBUG] get_response: http://121.46.19.90/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170606/89093558_3736315_tv_S_114528_26293/89093558_3736315_tv_S_114528_26293_005.mp4&new=/81/66/ZASWIn9yJS9knTMGYkilIK.mp4&key=eCZlMeLFijnnrdXXdrvoWnaSwF_Sp6JB&vid=3736316&uid=1499605810816&t=0.2822450397680005&rb=1
[DEBUG] get_response: http://121.46.19.90/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170606/89093558_3736315_tv_S_114528_26293/89093558_3736315_tv_S_114528_26293_006.mp4&new=/31/161/VQTBo0pIQRWDUTdys8hOdC.mp4&key=sS-xkTZLOQhVdYSnvV4b7mJPaUjTGANs&vid=3736316&uid=1499605810924&t=0.2010802982449209&rb=1
[DEBUG] get_response: http://121.46.19.90/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170606/89093558_3736315_tv_S_114528_26293/89093558_3736315_tv_S_114528_26293_007.mp4&new=/15/85/b8mHODempDTBg8lGd8WiPf.mp4&key=5jNXElwiy0acJgWmvX-s_-WcNntr6AzP&vid=3736316&uid=1499605811997&t=0.029607561506383218&rb=1
[DEBUG] get_response: http://121.46.19.90/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170606/89093558_3736315_tv_S_114528_26293/89093558_3736315_tv_S_114528_26293_008.mp4&new=/126/225/QL8St8VyEe6i94oVZ4zEwW.mp4&key=OSPSid0HMw75KsQ4LDA4LHre_K8Erwyu&vid=3736316&uid=1499605812072&t=0.5778062009726915&rb=1
[DEBUG] get_response: http://121.46.19.90/?prot=9&prod=flash&pt=1&file=/tv/Sample1/BackUp_Sample1/svc/20170606/89093558_3736315_tv_S_114528_26293/89093558_3736315_tv_S_114528_26293_009.mp4&new=/71/122/6szyjUu5Rs2I9MR0Y6FbwG.mp4&key=aNjK4rvdvtCmW3E79Qf9uImAIha47gF5&vid=3736316&uid=1499605812145&t=0.18693324951330492&rb=1
Site:       Sohu.com
Title:      欢乐颂2第6集
Type:       MPEG-4 video (video/mp4)
Size:       313.25 MiB (328466688 Bytes)

Downloading 欢乐颂2第6集.mp4 ...
 100% (313.2/313.3MB) ├██████████████████████████████████████████████┤[9/9]    3 MB/s
Merging video parts... ffmpeg version 3.3.1 Copyright (c) 2000-2017 the FFmpeg developers
```
